lenses_python/KerberosTicket.py

```diff
@@ -11,7 +12,8 @@ def __init__(self, service):
         kerberos.authGSSClientStep(krb_context, "")
         self._krb_context = krb_context
         # Create the value of authentication header
-        self.auth_header = ("Negotiate " +kerberos.authGSSClientResponse(krb_context))
+        self.auth_header = ("Negotiate " + kerberos.authGSSClientResponse(krb_context))
```

Comment:
use string interpolation

Reply:
We will move to string interpolation on a new branch, since this needs much work (changing almost all files :P)